### PR TITLE
CY-1060 Add Plugins Update API docs

### PR DESCRIPTION
**Pull request by davidginzbourg**

### CODE REVIEW

File: content/_plugins.md [hunk 1]
```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
...
 A `Plugin` resource.
+
+## The Plugins Update Resource
+
+### Attributes:
+
+Attribute | Type | Description
+--------- | ------- | -------
+`id` | string | A unique identifier for the plugins update.
+`state` | string | The state of this update (successful, failed, updating, etc...).
```
Comment (tyacbovi) [inline content/_plugins.md]: better to list all options to be on the clear side

File: content/_plugins.md [hunk 2]
```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
...
+`blueprint_id` | string | The id of the blueprint to its deployments.
```
Comment (tyacbovi) [inline content/_plugins.md]: The id of the blueprint which all of it's deployments will be updated.
Or something like that

File: content/_plugins.md [hunk 3]
```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
...
+`temp_blueprint_id` | string | The id of the temporary blueprint created for the purpose of this plugins update.
```
Comment (tyacbovi) [inline content/_plugins.md]: "The id of the temporary internal ...."

File: content/_plugins.md [hunk 4]
```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
...
+`execution_id` | string | The id of the execution performing the update.
```
Comment (tyacbovi) [inline content/_plugins.md]: The id of the update execution

File: content/_plugins.md [hunk 5]
```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
...
+`deployments_to_update` | list | A list of deployment updates IDs that are updated in this plugins update.
```
Comment (tyacbovi) [inline content/_plugins.md]: A list of deployment IDs that are updated in this plugins update

File: content/_plugins.md [hunk 6]
```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
...
+`forced` | bool | Whether or not this plugins update is forced.
```
Comment (tyacbovi) [inline content/_plugins.md]: Better to explain what is "force" in this context

File: content/_plugins.md [hunk 7]
```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
...
+ "resource_availability": "tenant",
+ "id": "ce63a9d6-726b-4ff2-a267-f953af5dc32d",
+ "execution_id": "8ec77e89-b418-4472-9b44-fe9d7a7fe163"
+}
+
+```
+
+`PUT "<manager-ip>/api/v3.1/plugins-updates/<blueprint-id>/update/initiate"`
+
+Update the plugins for that blueprint's deployments. **Supported for Cloudify Manager 5.0 and above.**
```
Comment (tyacbovi) [inline content/_plugins.md]: 5.0.0

File: content/_plugins.md [hunk 8]
```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
...
+
+### URI Parameters
+* `blueprint-id`: The id of the blueprint to update its deployments.
```
Comment (tyacbovi) [inline content/_plugins.md]: see previous note

File: content/_plugins.md [hunk 9]
```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
...
+      "offset": 0,
+      "size": 0
+    }
+  }
+}
+```
+
+`GET "{manager-ip}/api/v3.1/plugins-updates"`
+
+Lists the plugins updates. **Supported for Cloudify Manager 5.0 and above.**
```
Comment (tyacbovi) [inline content/_plugins.md]: 5.0.0

Comment (tyacbovi) [inline content/_plugins.md]: What this list includes in short, something like status and etc

File: content/_plugins.md (hunk 6 above)
Comment (davidginzbourg) [reply content/_plugins.md]: > Whether or not this plugins update was executed regardless of other active/failed plugins updates.
Better?

File: content/_plugins.md (hunk 9 above)
Comment (davidginzbourg) [reply content/_plugins.md]: It's written below in the "Response" header

File: content/_plugins.md (hunk 6 above)
Comment (tyacbovi) [reply content/_plugins.md]: Specifies whether to force the plugin update regardless of other active/failed plugins updates state.

**Pull request merged**